docs/learn/documentation/versioned/jobs/configuration-table.html

```diff
@@ -683,7 +689,7 @@
                 </tr>
 
                 <tr>
-                    <th colspan="3" class="section" id="streams"><a href="../container/streams.html">Systems (input and output streams)</a></th>
+                    <th colspan="3" class="section" id="systems"><a href="../container/streams.html">Systems (input and output streams)</a></th>
```

Comment:
Good catch. That was a mistake. 

I feel like we need a different doc to talk specifically about the system abstraction and this config section should link to that. I could file a JIRA for that if you agree.